src/Content/Item.php

```diff
@@ -494,21 +494,25 @@ public function visibleActivity($item) {
 		return true;
 	}
 
-	public function expandTags(array $item)
+	public function expandTags(array $item, bool $setPermissions = false)
```

Comment:
Please add documentation for this parameter, and consider renaming it to have something to do with "private" permissions.